packages/experiments/src/components/FloatingSuggestions/FloatingSuggestions.tsx

```diff
@@ -0,0 +1,352 @@
+import * as React from 'react';
+import * as styles from './FloatingSuggestions.scss';
+import { BaseComponent, css, KeyCodes } from 'office-ui-fabric-react/lib/Utilities';
+import { Callout, DirectionalHint } from 'office-ui-fabric-react/lib/Callout';
+import { IFloatingSuggestions, IFloatingSuggestionsProps, IFloatingSuggestionsInnerSuggestionProps } from './FloatingSuggestions.types';
+import { ISuggestionModel } from 'office-ui-fabric-react/lib/Pickers';
+import { ISuggestionsControlProps } from './Suggestions/Suggestions.types';
+import { SuggestionsControl } from './Suggestions/SuggestionsControl';
+import { SuggestionsStore } from './Suggestions/SuggestionsStore';
+
+export interface IFloatingSuggestionsState {
+  queryString: string;
+  suggestionsVisible?: boolean;
+  didBind: boolean;
+}
+
+export class FloatingSuggestions<TItem> extends BaseComponent<IFloatingSuggestionsProps<TItem>, IFloatingSuggestionsState>
+  implements IFloatingSuggestions<TItem> {
+  protected selection: Selection;
+
+  protected root = React.createRef<HTMLDivElement>();
+  protected suggestionStore: SuggestionsStore<TItem>;
+  protected suggestionsControl: React.RefObject<SuggestionsControl<TItem>> = React.createRef();
+  protected SuggestionsControlOfProperType: new (props: ISuggestionsControlProps<TItem>) => SuggestionsControl<TItem> = SuggestionsControl;
```

Comment:
I believe in the latest versions of react that we don't actually need this any more. You can do inline types like `<SuggestionsItem <TItem> ...>`
That said, I don't see this used anywhere, did you mean to leave it?